packages/tools/src/eventListeners/mouse/mouseDownListener.ts

```diff
@@ -144,7 +144,7 @@ function mouseDownListener(evt: MouseEvent) {
 
   state.element = <HTMLDivElement>evt.currentTarget;
 
-  state.mouseButton = evt.button;
+  state.mouseButton = evt.buttons;
 
```

Comment:
Changing this from `button` to `buttons` (on the surface) feels problematic because don't all of the `MouseBindings` enum values relate to the old `button` values? If so, please revert this back to `button`.

Reply:
nevermind... I was wrong

Reply:
Despite the fact I was wrong, I do not understand how the bindings were actually functional before this change... likely because the original event is stored in `eventDetail`. That said, it bothers me that `buttons` is now being assigned to the singular `mouseButton`. @sedghi any thoughts?

Reply:
I just checked and the `modifierkeys` example works even with the former/old code (and the rest of you other changes of course), so what is this particular change for actually? I apologize if these are stupid questions.

Reply:
The modifierKeys only worked for single modifiers, not combined modifiers, so this fix addresses that issue.
The bindings is somewhat problematic - I'm not sure why the state setup is there exactly.

Reply:
To clarify, the `modifierkeys` example still works if I revert line 147. So my question is why do we need the change for line 147. I understand we need all the other changes for combined modifiers.

Reply:
That allows the event to support multiple buttons, whereas using the .button doesn't allow for that.

Reply:
I understand. To clarify again, `state.mouseButton` appears (historically) to be for 'button` and the `buttons` value can be fetched from `eventDetail.event.buttons`. So either do not make this change, or rename `state.mouseButton` to `state.mouseButtons` so that there is no ambiguity. I recommend NOT making the change in case somebody, somewhere in the world is relying on it being `button`.

Reply:
I guess it is not that bad, adding another property to state is an overkill for this imo